access/two_factor_authentication.md

```diff
@@ -13,15 +13,15 @@ This policy applies to all Lullabot employees and contractors.
+Traditional authentication consists of a username and password. However, there is a more secure way to authenticate, called two-factor authentication. Two-factor authentication consists of something you know (a username and password) combined with a code available from something you have. The "something you have" is usually a single use PIN.
 
 When logging into an account that has been set up with two factor authentication, you will be asked for your username and password as usual, but you will also be asked for the _current_ PIN. 
 
-There are several ways to generate this PIN. We strongly recommend using 1Password's [built-in 2FA functionality](https://support.1password.com/one-time-passwords/). When an authentication app is linked to your account, the app will display a six digit PIN that changes once every minute. This is preferred over using SMS for 2FA, which can by [bypassed](https://www.securityweek.com/6-ways-attackers-are-still-bypassing-sms-2-factor-authentication/).
+There are several ways to generate this PIN. We strongly recommend using 1Password's [built-in 2FA functionality](https://support.1password.com/one-time-passwords/). When an authentication app is linked to your account, the app will display a PIN, commonly 6 digits, that changes once every minute or so. This is preferred over using SMS for 2FA, which can by [bypassed](https://www.securityweek.com/6-ways-attackers-are-still-bypassing-sms-2-factor-authentication/).
 
 Two-factor authentication is significantly more secure than single-factor (username and password only). If someone figures out your single-factor authentication credentials, they can access your account indefinitely without you knowing, or worse, log in and change your password to lock you out. With two-factor authentication, there is a time-based token that they would need to know, too, and it continuously changes. This keeps your account much more secure.
 
-You should be sure you have some way to recover access to your account if your phone is lost or destroyed. Most Lullabot-run accounts can be reset by an admin, but it can be problematic for personal accounts or smaller services without group or company features. An easy way to think about it is to think about how you would recover from a (local) disaster like a fire or flood. If you lost every piece of technology you owned, could you recover your accounts? Some strategies include:
+You should be sure you have some way to recover access to your account if your phone is lost or destroyed. Most Lullabot-run accounts can be reset by an admin, but it can be problematic for personal accounts or smaller services without group or company features. One way to think about it is to think about how you would recover from a (local) disaster like a fire or flood. If you lost every piece of technology you owned, could you recover your accounts? Some strategies include:
```

Comment:
"easy" is a judgement, and it might make people less likely to ask questions if they think they should have understood cause it was "easy".